utils/tree/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2017 OpenWrt.org
```

Comment:
Have you affiliated anyhow with OpenWrt? It would be best if you kept it as it was.

Reply:
Done.